test/suites/logscan/log-scan.robot

```diff
@@ -40,21 +40,25 @@ Should Find MicroShift Is Ready
     [Tags]    robot:exclude
     Pattern Should Appear In Log Output    ${CURSOR}    MICROSHIFT READY
 
-
 *** Keywords ***
 Setup
     [Documentation]    Test suite setup
     Check Required Env Variables
     Login MicroShift Host
-    Setup Kubeconfig    # for readiness checks
+    Stop MicroShift
+    Cleanup MicroShift    --all    --keep-images
     # Save the journal cursor then restart MicroShift so we capture
     # the shutdown messages and startup messages.
     ${cursor}=    Get Journal Cursor
     Set Suite Variable    \${CURSOR}    ${cursor}
-    Restart MicroShift
+    Setup Kubeconfig    # for readiness checks
+    Start MicroShift
+    Wait For MicroShift Service
     Restart Greenboot And Wait For Success
+    Stop MicroShift
```

Comment:
The previous implementation left the service running after Setup. This doesn't. We've seen issues with restarting the service, so maybe instead of stopping here it should restart so we continue to monitor that case?